packages/dashboard-frontend/src/services/workspace-adapter/index.ts

```diff
@@ -215,7 +215,7 @@ export class WorkspaceAdapter<T extends che.Workspace | devfileApi.DevWorkspace>
     if (isCheWorkspace(this.workspace)) {
       return attributesToType(this.workspace.devfile.attributes);
     } else {
-      console.error('Not implemented: get storage type of the devworkspace.');
+      // TODO add implementation for getting storage type of the devworkspaces
```

Comment:
I think console.error is OK at this phase.

Reply:
In this case, we will have 7 errors in the console just after reloading a workspace-detail page.
But ok. I have returned it.